recipes/eigen/bld.bat

```diff
@@ -0,0 +1,6 @@
+mkdir -p build
+cd build
+
+cmake -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_BUILD_TYPE=Release ..
+cmake --build . --target INSTALL --config Release
```

Comment:
Correct, tests must be built, but `ctest` should do that, no? Admittedly, there may be a bit of configuration needed here. Honestly, I almost question doing tests here at all given no libraries are distributed. Though making sure things were defined correctly is important.